docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<ul><li>the convention to have target before source is a bit unusual, maybe the other way round (does not change much, because it's symmetric, but agrees better with the usual convention.</li><li>given the goal to have a very compact NB, not sure this wrapper is really needed. wouldn't it be easier to have for instance</li></ul><p><code>grad_fn = jax.grad(lambda x, y : sinkhorn.solve(PointCloud(x, y)).reg_ot_cost</code>  in that sense it does not really matter if we return both gradients.</p><ul><li>Let's avoid using capital letters for variable names (naming convention of google python style)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Thanks ! </p><p> </p><p>About the wrapper, I feel like it is much easier to read like this, right? the lambda etc might be a bit too much for a new user?</p>

Reply:
<p>Also I reverted the names to <code>x</code> and <code>y</code>  like in the pointsclouds notebook, let me know if <code>x_source</code> / <code>x_target</code> are better names.   </p>

Reply:
<p>maybe lambda is a bit heavy, but </p><ul><li><code>cost_from_source</code> is not a very convincing function name (it's too long). </li><li>I don't think the closure on <code>y</code> is needed. It's ok to keep the <code>def ...</code>  if you prefer to avoid the lambda, but adding <code>y</code> to the list of arguments won't make any difference (<code>jax.grad</code> has <code>arg_nums</code> to 0 by default) and I think it's more simple. </li></ul>